website/source/docs/apps/index.html.md

```diff
@@ -5,7 +5,7 @@ sidebar_current: "docs-apps"
 description: |-
   The application type in the Appfile
   is the most important piece of information that tells Otto how to function.
-  Otto uses this information to know how to develop, build, and deploy
```

Comment:
- In serious: https://thewritepractice.com/why-you-need-to-be-using-oxford-commas/
- In fun: http://www.buzzfeed.com/adamdavis/the-oxford-comma-is-extremely-important-and-everyone-should#.igWEQLE81